api/src/main/java/io/minio/MinioClient.java

```diff
@@ -1938,7 +1940,8 @@ public void makeBucket(String bucketName, String region)
       configString = config.toString();
     }
 
-    executePut(bucketName, null, headerMap, null, US_EAST_1, configString, 0);
+    execute(Method.PUT, US_EAST_1, bucketName, null, headerMap, null,
+            "application/octet-stream", configString, 0);
```

Comment:
This is on purpose for `makeBucket()`  knows what to send there is no automatic region handling etc. So i see this is correct.